daemon/cluster/noderunner.go

```diff
@@ -14,6 +13,7 @@ import (
 	"github.com/docker/docker/daemon/cluster/executor/container"
 	lncluster "github.com/docker/libnetwork/cluster"
 	swarmapi "github.com/docker/swarmkit/api"
+	swarmallocator "github.com/docker/swarmkit/manager/allocator/cnmallocator"
```

Comment:
nit: was the alias needed here? (did it conflict, or was this just for readability?)

Reply:
for readability I added that like